featuretools/tests/entityset_tests/test_es.py

```diff
@@ -296,16 +373,20 @@ def test_converts_variable_types_on_init():
     assert isinstance(e['category_int'], variable_types.Categorical)
 
 
-def test_converts_variable_type_after_init():
-    df = pd.DataFrame({'id': [0, 1, 2],
-                       'category': ['a', 'b', 'a'],
-                       'ints': ['1', '2', '1']})
-
-    df["category"] = df["category"].astype("category")
-
+# TODO: Fails with dask, same issue as above, can't force to different vtype
+def test_converts_variable_type_after_init(df4):
```

Comment:
should we xfail this test and the one above?

Reply:
These actually don't fail since there was a simple fix to use `dd.to_numeric` instead of `pd.to_numeric` so the comment should just be removed